e2e/LongHaul/service/IotHub.cs

```diff
@@ -100,7 +100,6 @@ await foreach (ClientTwin deviceTwin in allDeviceTwins)
                             async Task OperateWithDeviceAsync()
                             {
                                 Logger deviceLogger = _logger.Clone();
-                                deviceLogger.LoggerContext.Add(DeviceId, deviceId);
```

Comment:
We already add logger context information in ctor() for DeviceOperations. This duplicate add() caused it to fail and deviceOperations never running.

Reply:
My mistake.